packages/blade/src/components/Form/Selector/SelectorInput.web.tsx

```diff
@@ -62,15 +62,16 @@ const StyledInput = styled.input<HoverProps>(({ theme, isChecked, isDisabled, ha
 
 const _SelectorInput: React.ForwardRefRenderFunction<
   BladeElementRef,
-  HoverProps & { inputProps: any }
-> = ({ inputProps, isChecked, isDisabled, hasError }, ref) => {
+  HoverProps & { inputProps: any; tabIndex?: number }
```

Comment:
since this is optional and if not set then it'll be undefined means it'll set to 0 which is the default behavior I'm assuming?

Reply:
It won't set to 0, It will be undefined no? it won't set `tabindex` prop in that case